app/assets/javascripts/admin/checkouts/edit_override.js

```diff
@@ -0,0 +1,53 @@
+$(document).ready(function() {
```

Comment:
Nice work @bingxie!

Could we turn this into a directive? You can use the `restrict: 'C'` option to bind a directive to an element with a particular class (ie. #customer_search_override could bind to a directive named customerSearchOverride). Then the rest of this function can just go inside the link function on the directive, using `element` instead of `$("#customer_search_override")` to make the `select2` call.

What do you think?